packages/server/src/controller/comment.js

```diff
@@ -387,10 +387,12 @@ module.exports = class extends BaseRest {
       data.status === 'approved' &&
       oldData.pid
     ) {
+      // FIXME: pid is not defined
       let pComment = await this.modelInstance.select({ objectId: pid });
       pComment = pComment[0];
 
       const notify = this.service('notify');
+      // FIXME: resp is not defined
       await notify.run(resp, pComment, true);
```

Comment:
This PR is not expected to pass CI, becuase there are issues here

Reply:
I fixed it, and ci seem works well.